### PR TITLE
Created Use Case document

### PR DESCRIPTION
**Pull request by bparish628**
This includes our use case of allowing a project lead to determine what libraries to use within the project.

### CODE REVIEW

File: docs/Use Case.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Use Case
+
+**Title:** Determining the health of a Github repository  
+
+**Primary Actor:** Project Leader 
+
+**Goal in Context:** The project lead is able to access if an open source project is healthly enough for its company's standards to incorporate into their product.
```
Comment (jakeharding) [inline docs/Use Case.md]: Is `access` supposed to be `assess`?

File: docs/Use Case.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Use Case
+
+**Title:** Determining the health of a Github repository  
+
+**Primary Actor:** Project Leader 
+
+**Goal in Context:** The project lead is able to access if an open source project is healthly enough for its company's standards to incorporate into their product.  
+
+**Stakeholders:**  
+* Project Lead: To understand if software is apporiate to incorporate into their own product
```
Comment (jakeharding) [inline docs/Use Case.md]: Misspelling of appropriate.

File: docs/Use Case.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* Developer: To clearly gauge if a piece of software will need to be maintained by them or if it's reliable
+* Manager: To access the longevity of this project with regards to dependencies
```
Comment (jakeharding) [inline docs/Use Case.md]: access == assess?

File: docs/Use Case.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+**Preconditions:**  
+* Metrics for a healthy repository are already defined
+* The repositories to be compared are already picked out 
+
+**Main Success Scenario:** Project lead understands the metrics presented and makes an accurate decision on if the software should be incoporated  
+
+**Failed End Conditions:** Project lead recieves little to no information on the health of the chosen repository.
```
Comment (jakeharding) [inline docs/Use Case.md]: Misspelling of receives.

File: docs/Use Case.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+**Stakeholders:**  
+* Project Lead: To understand if software is apporiate to incorporate into their own product
+* Developer: To clearly gauge if a piece of software will need to be maintained by them or if it's reliable
+* Manager: To access the longevity of this project with regards to dependencies
+
+**Preconditions:**  
+* Metrics for a healthy repository are already defined
+* The repositories to be compared are already picked out 
+
+**Main Success Scenario:** Project lead understands the metrics presented and makes an accurate decision on if the software should be incoporated
```
Comment (jakeharding) [inline docs/Use Case.md]: I suggest removing the word on between decision and if.

File: docs/Use Case.md (hunk 4 above)
Comment (jakeharding) [inline docs/Use Case.md]: Nitpick: Some of these are bullets and some are sentences. Consistently use periods at the end of the sentences and bullets can go either way as long as they are consistent.

File: docs/Use Case.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+**Preconditions:**  
+* Metrics for a healthy repository are already defined
+* The repositories to be compared are already picked out 
+
+**Main Success Scenario:** Project lead understands the metrics presented and makes an accurate decision on if the software should be incoporated  
+
+**Failed End Conditions:** Project lead recieves little to no information on the health of the chosen repository.  
+
+**Trigger:** Project lead is assigned to a brand new project and needs to access what libraries and dependencies should be used
```
Comment (jakeharding) [inline docs/Use Case.md]: access == assess?

**Review by jakeharding — CHANGES_REQUESTED** [review]
A few misspellings and some punctuation consistency.

**Comment by bparish628** [comment]
Well shoot. That's a lot. I really wish it showed me misspellings :cry:

**Review by jakeharding — APPROVED** [review]

**Pull request merged**